plugins/cisco_asa/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: cisco_asa
 title: Cisco Adaptive Security Appliance
 description: The Cisco ASA plugin allows you to automate the management of network objects
-version: 1.4.2
+version: 1.5.0
 vendor: rapid7
 support: community
 status: []
```

Comment:
Just below this, add a section for supported versions: https://docs.rapid7.com/insightconnect/plugin-spec